.circleci/config.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     steps:
       - checkout
       - run: go get -v -t -d ./...
+      - run: make test-unit
       - run: make test-unit-cover
       - run: make test-integration
```

Comment:
Can we put test unit cover after the test-integration ? If test-unit and test-integration passes, we know our PR is good already.

Reply:
It would need to be in a separate workflow, because our current status check against only allowing merges when `build` passes will require all steps to have finished regardless of order. (from #275)